duneclient/src/faction.js

```diff
@@ -15,7 +15,26 @@ class Faction extends React.Component {
 
         if (Array.isArray(this.props.factionstate.treachery)){
             return this.props.factionstate.treachery.map((name, i) => {
-                return <Card type="Treachery" key={i} name={name} />;
+                const interaction = this.props.interaction;
+                const mode = interaction.mode;
+
+                let onClick = null;
+                let selected = false;
+                if (name === "Cheap-Hero/Heroine") {
```

Comment:
Not the most into this.. other ideas?

Reply:
I don't know... seems okay to me. It's a weird case.